broker/startup_checks_test.go

```diff
@@ -136,6 +154,24 @@ var _ = Describe("Initializing the broker", func() {
 			Expect(brokerCreationErr).To(HaveOccurred())
 			Expect(brokerCreationErr).To(MatchError(`BOSH Director error: unrecognised BOSH Director version: "0000 (00000000)". ODB requires BOSH v257+.`))
 		})
+
+		Context("when CF startup checks are disabled", func() {
+			It("still verifies BOSH version", func() {
+				boshInfo := &boshdirector.Info{Version: "0000 (00000000)"}
+
+				_, brokerCreationErr := broker.New(
+					boshInfo,
+					boshClient,
+					cfClient,
+					serviceAdapter,
+					fakeDeployer,
+					serviceCatalog,
+					true,
+					loggerFactory,
+				)
+				Expect(brokerCreationErr).To(HaveOccurred())
+			})
+		})
```

Comment:
Same as above. I think it would be better to you avoid the `Context` and use `It` directly.